src/style.css

```diff
@@ -1 +1,16 @@
-/* styles go here */
+
+.block {
+  margin-bottom: 20px;
+}
+
+.fieldset {
+  padding-top: 10px;
+}
+
+.number {
+  width: 3.3em;
+}
+
+.selector {
+  overflow: hidden;
+}
```

Comment:
what about those classes, do we need to style them?)

Reply:
. selector, we don`t need it, but without .number i didn`t pass the test then